lib/screens/login/login_screen.dart

```diff
@@ -0,0 +1,80 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:lydiaryanfluttersurvey/gen/assets.gen.dart';
+import 'package:lydiaryanfluttersurvey/screens/widgets/login_input_widget.dart';
+
+import '../../resources/dimensions.dart';
+import '../widgets/rounded_rectangle_button_widget.dart';
+import 'login_keys.dart';
+
+class LoginScreen extends StatelessWidget {
+  const LoginScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.transparent,
+      body: Container(
```

Comment:
We will re-use this background image in the Reset Password screen, so what do you think about creating a background widget here? So we can re-use it 🙏

Reply:
@markgravity Let’s refactor it later when reusing it. I would like to close this milestone asap 🙏🏻